inference-engine/src/gna_plugin/backend/gna_limitations.hpp

```diff
@@ -31,7 +31,8 @@ inline bool IsTransposeSupported(const std::vector<size_t>& shape) {
     auto shape_no_1 = shape;
     shape_no_1.erase(std::remove(shape_no_1.begin(), shape_no_1.end(), 1), shape_no_1.end());
     if (shape_no_1.size() != 2) return false;
-    size_t min, max;
+    size_t min = 0;
```

Comment:
is there a possible case when min and max will be still uninitialized after std::tie(min, max) = ... construction?

Reply:
The static analyzer gives an error when using uninitialized min and max in std::tie(min, max) = ...